username.rb

```diff
@@ -2,13 +2,110 @@
 # Run `rspec /spec/username_spec.rb` to get started.
 
 def format_name(first, last)
-  nil
+  if last.length < 1 || first.length < 1
+    nil
+  else
+    first_name = ''
```

Comment:
This is cool but you're doing a lot of work here! I would think about how you could refactor this code to write it more simply.